### PR TITLE
Add Run On Device page

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**
https://docs.microej.com/en/feature-run-on-device/SDK6UserGuide/runOnDevice.html

### CODE REVIEW

File: SDK6UserGuide/runOnDevice.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. _sdk_6_run_on_device:
+
+Run on Device
+=============
+
+The SDK allows to deploy deploy an Application on a Device thanks to the Gradle ``runOnDevice`` task.
```
Comment (denisbedin) [inline SDK6UserGuide/runOnDevice.rst:6]:
```suggestion
The SDK allows to deploy an Application on a Device thanks to the Gradle ``runOnDevice`` task.
```

File: SDK6UserGuide/runOnDevice.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      .. image:: images/intellij-runOnDevice-gradle-project.png
+         :width: 30%
+         :align: center
+
+The build should be successful and the output should end with::
+
+   Execution of script '<RUN_SCRIPT_PATH>' done.
+   BUILD SUCCESSFUL
+
+where ``RUN_SCRIPT_PATH`` is the absolute path to the ``run.[sh|bat]`` scripts of the VEE Port.
```
Comment (denisbedin) [inline SDK6UserGuide/runOnDevice.rst:49]:
```suggestion
where ``RUN_SCRIPT_PATH`` is the absolute path to the ``run.[sh|bat]`` script of the VEE Port.
```

Comment (tdelhomenie-microej) [reply SDK6UserGuide/runOnDevice.rst]: Done.

File: SDK6UserGuide/runOnDevice.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/runOnDevice.rst]: Done.

**Review by denisbedin — APPROVED** [review]

**Pull request merged**